cmd/backend/start/start.go

```diff
@@ -14,15 +12,13 @@ import (
 var StartCmd = &cobra.Command{
 	Use:   "start",
 	Short: "Starts the training backend",
-	Long:  `Starts an instance of the training backend Django app in /training in the terminal`,
+	Long:  `Starts an instance of the training backend Django app in /training in the terminal. To change the backend port, set the environment var BACKEND_PORT. `,
```

Comment:
so would the invocation still be `dlp-cli backend start`?

Reply:
yes